### ISSUE TITLE
Extension will work only with english locale

### ISSUE THREAD

**Issue by ghost** [mattsimonis/meet-mute#1]
Hi,

current mute button selector looks for "microphone" in label, which breaks this plugin if googles active language is set to anything else but english. I had a look into element and indeed it does not look trivial to get it without looking at label:

```
<div jsshadow="" role="button" class="U26fgb JRY2Pb mUbCce kpROve uJNmj FTMc0c N2RpBe jY9Dbb" jscontroller="lCGUBd" jsaction="click:cOuCgd; mousedown:UX7yZ; mouseup:lbsD7e; mouseenter:tfO1Yc; mouseleave:JywGue; focus:AHmuwe; blur:O22p3e; contextmenu:mg9Pef;touchstart:p6p2H; touchmove:FwuNnf; touchend:yfqBxc(preventMouseEvents=true|preventDefault=true); touchcancel:JMtRjd" jsname="BOHaEe" aria-disabled="false" tabindex="0" data-response-delay-ms="250" data-tooltip-position="top" data-tooltip-vertical-offset="12" aria-label="Turn on microphone (⌘ + D)" data-tooltip="Turn on microphone (⌘ + D)" data-is-muted="true">
```

Do you think there is another way to find mute button?

**Comment by introkun** [comment]
Fixed by my PR https://github.com/mattsimonis/meet-mute/pull/8

**Comment by mattsimonis** [comment]
Fixed by #8. Closing.
